src/api/v3/feedback/__init__.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from django.http import HttpResponse
+
+
+def feedback_handler(func):
+    def handle_feedback(request, region_slug, language_code):
+        if request.method != 'POST':
+            return HttpResponse('Invalid request.', status=405)
+        data = json.loads(request.body)
+
+        comment = data.pop("comment", "")
```

Comment:
Just a hint: `request.POST` is immutable, so we have to use `request.POST.get()` instead of  `request.POST.pop()`.

Reply:
At the end of this decorator function, `data` should be a dictionary with all params but without comment, rating and category.
Online I found
```
request.GET
data = request.GET.copy()
data.pop('key') 
```
But that doesn't work. For a strange reason, a.get() returns the desired object, but a.pop() returns an array with this object.

Reply:
The page feedback endpoint uses `json.loads(request.body)`.
Is that an error?

Reply:
Ok that seems strange, whenever I used `request.GET.copy()`, it worked without problems. Are you sure it returned a list or dict? Because [the official python documentation](https://docs.python.org/3/library/stdtypes.html#dict.pop) does not describe a possibility how `pop()` should return anything other than a single value... (except for the case that this value is in fact a list :sweat_smile:)

The page feedback endpoint was implemented during the expedia hackathon, where we didn't have much time for in-depth code reviews. It turns out, that we missed some error there.

Reply:
Ah, `request.GET/request.POST` isn't a dict but a `django.http.QueryDict`. Apparently, it is possible for such an object to contain multiple elements with the same key.
If a key exists multiple times, `get()` returns an arbitrary object, but `pop()` returns a  list of all objects :laughing: 
https://docs.djangoproject.com/en/3.0/ref/request-response/#django.http.QueryDict.pop

So, I think `data.pop('key')[0]` with a check if the array contains at least one element should work.

Reply:
Ah, interesting! Yes, that should work!

Reply:
`request.POST.copy().dict()` returns the QueryDict as a normal dict (eliminating multiple keys by choosing one arbitrary value). I think this is the best way.